test/e2e_node/conformance/run_test.sh

```diff
@@ -0,0 +1,171 @@
+#!/bin/bash
```

Comment:
I don't think we should manage kubelet lifetime with a bash script or in the test suite.

For now, the script is only meant to demonstrate the usage of conformance test.
I hope we'll have a standard kubelet launcher, maybe with systemd or supervisord. Here we just need to call the launcher and run test.